Dockerfile

```diff
@@ -1,36 +1,9 @@
 # syntax = docker/dockerfile:1.1-experimental
 FROM ubuntu:focal
+WORKDIR /project
 
```

Comment:
We need to `apt-get install -y ca-certificates` for the bucket and HTTP source connections to work.